internal/client/src/client/client_test.go

```diff
@@ -507,3 +510,61 @@ func TestUpdateWallet(t *testing.T) {
 		t.Errorf("private keys do not match: %v != %v", encPrivActual, encPrivExpected)
 	}
 }
+
+func TestRequestWalletInfo(t *testing.T) {
+	if err := SetupWallet(); err != nil {
+		t.Errorf("failed to setup wallet:\n%s", err.Error())
+	}
+	defer func() {
+		if err := os.Remove("aurum_wallet.json"); err != nil {
+			t.Errorf("failed to remove aurum_wallet.json:\n%s", err.Error())
+		}
+	}()
+
+	dbName := "accounts.tab"
+	dbc, _ := sql.Open("sqlite3", dbName)
+	defer func() {
+		err := dbc.Close()
+		if err != nil {
+			t.Errorf("Failed to close database: %s", err)
+		}
+		err = os.Remove(dbName)
+		if err != nil {
+			t.Errorf("Failed to remove database: %s", err)
+		}
+	}()
+	_, err := dbc.Exec("CREATE TABLE IF NOT EXISTS account_balances (public_key_hash TEXT, balance INTEGER, nonce INTEGER)")
+	if err != nil {
+		t.Errorf("Failed to create table in database: %s", err)
+	}
+
+	walletAddress, err := GetWalletAddress()
+	if err != nil {
+		t.Errorf("failed to retrieve wallet address:\n%s", err.Error())
+	}
+
```

Comment:
Can you add code to test the case where the wallet address does not exist in the table? It would be before the insertion. You can probably implement a `for` loop and just reuse the code.

Reply:
I only see one call to `RequestWalletInfo`. What I meant was that there should be a case where you call `RequestWalletInfo` and the result from the producer should be a "not found" response.

Reply:
- [ ] Case where wallet address does not exist should return an `error`
- [ ] Case where wallet address does exist should return `nil`